highway.tex

```diff
@@ -24,807 +26,1323 @@
 
-Bitcoin \cite{nakamoto2008bitcoin} introduced a \emph{permissionless} Proof-of-Work consensus algorithm, where the set of participating nodes is continually changing, and anyone can join or leave at any time, and Ethereum \cite{wood2014ethereum} extended this concept from a pure cryptocurrency to a general programmable smart contract platform. Both networks have very high value and correspondingly high requirements for security, due to potentially large incentives for attackers.
 
-Proof-of-Work inherently has extremely high energy requirements \cite{de2018bitcoin}. Proof-of-Stake networks (e.g. \cite{kiayias2017ouroboros}) attempt to address this issue by giving the power to make consensus decisions not to the nodes that contribute the most computing resources, but to the ones that deposit the highest stakes as a collateral. They aspire to provide the same or higher levels of security and performance, while requiring less energy by many orders of magnitude.
 
 
-\subsection*{Our Contribution}
+\section{Our Results}
 
-Highway is a concrete consensus algorithm based on CBC Casper \cite{zamfir2018casper} together with a criterion for finality, both proposed by Daniel Kane. It is live in a partially synchronous network, if the participants have clocks that are synchronized up to an unknown but bounded difference.
+\subsection{Model}
 
-At the heart of Casper we made two design choices that keep it simple and intuitive:
+We consider a system with a fixed set $\mathcal{V}$ of $n$ validators, each of them equipped with public key that is known to other nodes.
+This model matches the scenario of 'permissioned blockchain', but the protocol can be applied to semi-permissionless scenario as well by rotating the set of validators.
+Our model makes the following assumptions:
 \begin{itemize}
-  \item There are no targeted messages sent only to specific nodes, like e.g. \cite{miller2016honey}. Instead, all messages are ``broadcasted'' to all network nodes and they attest to earlier messages; all nodes see the same, continuously growing graph of messages, similar to \cite{baird2016hashgraph,chevalier2018parsec,gkagol2019aleph,moser1999byzantine}.
-  \item Decisions are made using simple, explicit voting: You always have to follow the plurality.\footnote{This is in contrast to protocols like \cite{buchman2018latest,castro1999practical,muratov2018yac} where a leader proposes and if not enough confirmations are collected, a new leader retries with a different proposal, or e.g. \cite{kiayias2017ouroboros}, where the longest chain is always selected.}
+    \item\emph{(Reliable point-to-point communication)}
+    We assume that channels do not drop messages and all messages in the protocol are authenticated by digital signature of the sender.
+    % Validator receives a message from another validator if and only if the latter sent it and intended it to be received by the former. 
```

Comment:
Maybe remove this comment; seems to be a reformulation of the previous statement.

Reply:
Yeah, it's the previous version of that statement. Removed now